### PR TITLE
Add Python3 support

### PR DESCRIPTION
**Pull request by zakj**

### CODE REVIEW

File: test_mixpanel.py [hunk 1]
```diff
@@ -26,11 +28,12 @@ def send(self, endpoint, event, api_key=None):
...
+    if isinstance(s, six.binary_type):
+        s = s.decode('utf8')
     blob = cgi.parse_qs(s)
-    if 'data' in blob:
-        if len(blob['data']) != 1:
-            pytest.fail('found multi-item data: %s' % blob['data'])
-        blob['data'] = json.loads(base64.b64decode(blob['data'][0]))
+    if len(blob['data']) != 1:
+        pytest.fail('found multi-item data: %s' % blob['data'])
+    blob['data'] = json.loads(base64.b64decode(blob['data'][0]).decode('utf8'))
```
Comment (joeatwork) [inline test_mixpanel.py]: This line is (still) pretty dense

**Comment by joeatwork** [comment]
Great news! This is a very exciting pull request!

**Comment by joeatwork** [comment]
We may want to document the dependency (or even vendor it) - it might be enough to print a descriptive error if we fail to import six

File: test_mixpanel.py [hunk 2]
```diff
@@ -26,11 +28,13 @@ def send(self, endpoint, event, api_key=None):
...
+        s = s.decode('utf8')
     blob = cgi.parse_qs(s)
-    if 'data' in blob:
-        if len(blob['data']) != 1:
-            pytest.fail('found multi-item data: %s' % blob['data'])
-        blob['data'] = json.loads(base64.b64decode(blob['data'][0]))
+    if len(blob['data']) != 1:
+        pytest.fail('found multi-item data: %s' % blob['data'])
+    json_bytes = base64.b64decode(blob['data'][0])
+    blob['data'] = json.loads(json_bytes.decode('utf8'))
```
Comment (zakj) [inline test_mixpanel.py]: @joeatwork Split into two lines here, which hopefully makes it at least slightly clearer. There's a lot of en/de-coding to do!

**Comment by zakj** [comment]
I added an `install_requires` argument in `setup.py`, which does the right thing when installing via `easy_install` or `pip`. Can we assume that anyone installing via some other method should be able to understand Python's `ImportError`?

**Comment by joeatwork** [comment]
That seems sufficient- At this point, it's possible that people are just downloading __init__.py and doing stuff with it, but maybe that's not a UX we want to lose a lot of sleep over supporting :)

**Pull request merged**